tests/general/InvalidLinkBearTest.py

```diff
@@ -253,6 +255,11 @@ def test_redirect_threshold(self):
             self.check_validity(self.uut, long_url_redirect,
                                 settings={'follow_redirects': 'true'})
 
+            with prepare_file(short_url_redirect, None) as (lines, _):
```

Comment:
why do you need `prepare_file`?

Reply:
Actually I just need to add a linebreak (`\n`) to each elements, since `LocalBearTestHelper.get_results` uses `prepare_file` to add the linebreaks to the lines before inputting it to `bear.execute`.

If I just use `short_url_redirect` directly as an argument for instancing `Diff()`, it will raise `AssertionError`, since the bear actually is processing the lines with the linebreaks added